modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/scripttask/ScriptTaskTest.java

```diff
@@ -39,84 +43,63 @@ public void testSetScriptResultToProcessVariable() {
 
         ProcessInstance pi = runtimeService.startProcessInstanceByKey("setScriptResultToProcessVariable", variables);
 
-        assertEquals("hello", runtimeService.getVariable(pi.getId(), "existingProcessVariableName"));
-        assertEquals(pi.getId(), runtimeService.getVariable(pi.getId(), "newProcessVariableName"));
+        assertThat(runtimeService.getVariable(pi.getId(), "existingProcessVariableName")).isEqualTo("hello");
+        assertThat(runtimeService.getVariable(pi.getId(), "newProcessVariableName")).isEqualTo(pi.getId());
     }
 
     @Test
     @Deployment
     public void testFailingScript() {
-        Exception expectedException = null;
-        try {
-            runtimeService.startProcessInstanceByKey("failingScript");
-        } catch (Exception e) {
-            expectedException = e;
-        }
-
-        // Check if correct exception is found in the stacktrace
-        verifyExceptionInStacktrace(expectedException, MissingPropertyException.class);
+        assertThatThrownBy(() -> runtimeService.startProcessInstanceByKey("failingScript"))
+                .hasStackTraceContaining("MissingPropertyException");
```

Comment:
Although this is correct. I would prefer to keep the same assertion as before. Verify that the `MissingPropertyException` is in the cause is one of the causes in the stacktrace and not just containing the string

Reply:
@filiphr OK; fixed.

Reply:
Thanks for both fixes